### DISCUSSION TITLE
No repsonse to my GitHub support ticket after 3 weeks - affecting my work.

### DISCUSSION THREAD

**Discussion by seailz** [community/community#129217] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I've been waiting 3 weeks for a response to a GitHub support ticket relating to an organization of mine that's been flagged. It's completely unacceptable as it's affecting my work and I'm unable to get things done.

**Comment by markfilipak-windows** [comment]
seailz, I have no help for you. I, myself am a lost person here. My repo -- I think it's called "repo" -- has disappeared. I wish you luck, because it appears that luck is what makes a difference. If I could think of a way we could help each other, I'd help. Good Luck
--Mark Filipak.

**Comment by Honsal** [comment]
Welcome my friend, I am not organization but symptom is same. Suddenly my account has been flagged maybe because of hacker's activity or something, and supports are not responding over 3 weeks.

Even worse thing is, my monthly copilot subscription has paid after flagged. But I can't use copilot because my account is flagged. (Forbidden response)

And I can't get refund or recovery because they're not responding. Ridiculous.

See - [My Discussion(129364)](https://github.com/orgs/community/discussions/129364)

**Comment by vitytradez** [comment]
It has been almost a month that my support ticket has been open, no response yet. Unfortunately it has left a bit of a sour taste for me as I was very excited to finally be making enough progress as a developer to create projects worth sharing.

**Comment by sahaj-96** [comment]
Hiiii @seailz seems like your issue is resolved. i faced a similar issue today afternoon. Can u pls tell how much time it took for Github Support to respond?

**Reply by HimanshuMohanty-Git24** [reply]
Hi @sahaj-96 I think the issue got solved for you can you guide me how it got solved ? i am not able to contribute as well as share my repos to others as my account is flagged whenever someone goes to my account 404 error is shown:
![image](https://github.com/community/community/assets/94133298/fe15c1d9-3e12-492d-9bf3-7c8b47378f85)

**Comment by Dimash357** [comment]
response 2

**Reply by HimanshuMohanty-Git24** [reply]
I guess i have to just wait for my luck 🥲
